src/clients/BundleDataClient/BundleDataClient.ts

```diff
@@ -431,52 +438,6 @@ export class BundleDataClient {
       }, toBN(0));
   }
 
-  private async getLatestProposedBundleData(): Promise<{ bundleData: LoadDataReturnValue; blockRanges: number[][] }> {
```

Comment:
We no longer need these functions in https://github.com/across-protocol/relayer/pull/2085